CONTRIBUTING.md

```diff
@@ -78,5 +79,17 @@ Once your changes and tests are ready to submit for review:
     Note that squashing at the end of the review process should not be done. Rather, it should be done when the pull request is [integrated
     via GitHub](https://github.com/blog/2141-squash-your-commits). 
 
+### Reviewer Responsibilities
+A good code review considers many aspects of the PR
+- Will the code work and will it work in the **long-term**?
+- Is the code undertandable and readable? Is documentation needed? 
+- What are the security implications of the PR?
```

Comment:
Security implications are the responsibility of both the developer and the reviewer.  And a checklist is needed for the things to look for.